README.md

```diff
@@ -4,21 +4,44 @@
 
 > This JavaScript capstone project is built on the [TVMAZE API](https://static.tvmaze.com/apidoc/), an external API. The cards display the movies information that was retrieved from the API. The user has the option to like and comment on the movies cards.
 
+![](src/assets/cinema.JPG)
+
+![](src/assets/mobil.JPG)
+
 ## Built With
 
-- HTML, CSS, JavaScript, and API
+- HTML, CSS, JavaScript, and Tv-Maze API
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://natanima.github.io/Javascript-capstone-project/)
 
```

Comment:
- Hey, as it's a requirement that you deploy your app, the current deployment is not showing correctly. You can try using netlify. Please, kindly update this to make your readme more professional.

Reply:
# :point_right:  PRESENTATION VIDEO

- [ ] As it is a requirement that the video length should not exceed 5 min, Please, kindly update the video to meet this part of the project requirement as currently, the [video length](https://drive.google.com/file/d/1l3tjUHZ3Pg7TJk6XCkybGSyrFQJyvutS/view) is `6:18` which doesn't meet the project requirement.
- [ ] Also, kindly add the video to the readme file.

Reply:
Please It does not have a requirement for the length of the presentation video on the project @zieeco

Reply:
@NatanimA 
Please, kindly see the image below for clarification:


![Screenshot from 2022-10-20 17-20-32](https://user-images.githubusercontent.com/59397159/197012638-bad795b8-505c-4b55-b96d-26b3f6ff4577.png)

Reply:
The Link works @zieeco

Reply:
@NatanimA 

My apologies for not seeing the `npm test` part. I'd remove it.
For the deployment, at the time I tried the link, it wasn't showing correctly. Again, I apologize. I'd remove it now

Thanks for your understanding!

Reply:
Please note that I can not redo the presentation at this time It is late at night for me as well as my partner.